build.gradle

```diff
@@ -9,7 +9,7 @@ plugins {
 }
 
 group = 'uk.gov.hmcts.reform'
-version = '2.2.6'
+version = '2.3.0'
```

Comment:
let's do this 3.0.0 :)

Reply:
Yeah, that makes sense - the flip side of that though is that we are doing breaking changes here (what we have in 2.3.0 is not compatible with 2.2.6), so maybe a new major version would make sense.